platform/accessibility/guidance/staging-review-processes.md

```diff
@@ -334,4 +434,4 @@ Unresolved screen reader testing issues are documented in issue tickets and list
 *   [Cross-platform Screen Reader Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/resources/screenreader-testing-cross-platform.md)
 *   [WebAIM: Using NVDA to Evaluate Web Accessibility](https://webaim.org/articles/nvda/)
 *   [NVDA download](https://www.nvaccess.org/download/)
-*   [NVDA Focus Plugin](https://addons.nvda-project.org/addons/focusHighlight.en.html) 
+*   [NVDA Focus Plugin](https://addons.nvda-project.org/addons/focusHighlight.en.html) 
```

Comment:
This plugin is not compatible with the newest NVDA update. Let's remove it for now, and leave ourselves room to add it back later if it's fixed.